plenum/server/node.py

```diff
@@ -2311,16 +2402,18 @@ def verifySignature(self, msg):
         if not isinstance(req, Mapping):
             req = msg.as_dict
 
-        identifier = self.authNr(req).authenticate(req)
+        identifiers = self.authNr(req).authenticate(req)
         logger.debug("{} authenticated {} signature on {} request {}".
-                     format(self, identifier, typ, req['reqId']),
+                     format(self, identifiers, typ, req['reqId']),
                      extra={"cli": True,
                             "tags": ["node-msg-processing"]})
 
     def authNr(self, req):
         return self.clientAuthNr
 
     def isSignatureVerificationNeeded(self, msg: Any):
```

Comment:
Looks like this is not in sync with implementation in indy-node (that was removed in indy-node, so I expect that signature verification may not work correctly for query requests)

Reply:
I see it is there in indy-node's master branch, but query requests do not need signature verification anyway since anyone can query

Reply:
Yes, but I think if we don't check this and call `verifySignature` on query requests, then we will fail because of missing signature exception.